src/doc/rustc/src/platform-support/armv5te-none-eabi.md

```diff
@@ -0,0 +1,75 @@
+# `armv5te-none-eabi`
+
+**Tier: 3**
+
+Bare-metal target for any cpu in the ARMv5TE architecture family, supporting
+ARM/Thumb code interworking (aka `a32`/`t32`), with ARM code as the default code
+generation.
+
+The `thumbv5te-none-eabi` target is the same as this one, but with THUMB code as the default.
+
+In particular this supports the main CPU of the Nintendo DS, but there's nothing DS
```

Comment:
This sentence is probably superfluous/excessive for the documentation of the very generic baremetal target that `*-none-*` tend to be. There are a wide variety of applications in which v5te has been used (phones, PDAs, etc. and more recently a Kindle which is also a common homebrew/custom firmware target.)

Reply:
That's fair, the DS specific parts can be removed. I was mostly basing this off of `armv4t-none-eabi`, which makes particular note of the GBA.